custom.css

```diff
@@ -1,155 +1,306 @@
+/* Darius Dictionary Colorizer
+ * The latest version can be found at
+ * https://github.com/themoeway/yomichan-dict-css
+ */
+
+:root {
+  /* Change this opacity to 0% if you don't want backgrounds to change.
+   * Or increase it if you want them to change more.  
+   */
+  --dict-change-background-color-opacity: 5%;
```

Comment:
Hmm, I was quite particular about my choices of opacities and colors. I see that this makes it all 5% opacity. Did you make sure to confirm everything is visually balanced on both on light mode and dark mode, or was it just a "why not unify it" sort of decision?

Reply:
Honestly it was mostly a "why not unify it" decision. I didn't confirm the visuals in dark mode - it isn't officially supported, right? Though there are those dark mode variables in yomichan css you can use. 

I'll try to match the backgrounds more closely and add opacities for each background, that will be multiplied by the overall opacity value.

Reply:
It's basically officially supported. There's a toggle in the settings, and when I first released this CSS months ago the first piece of feedback I got was about dark mode.